client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -314,6 +319,18 @@ class BusinessDetails extends Component {
 					'woocommerce-admin'
 				),
 			},
+			{
+				slug: 'google-ads',
+				title: __(
+					'Drive sales with Google Shopping',
+					'woocommerce-admin'
+				),
+				icon: 'onboarding/google-ads.png',
```

Comment:
I think we may want to start hosting these images on woocommerce.wordpress.com.  See @timmyc's PR https://github.com/woocommerce/woocommerce-admin/pull/3719 for prior art.

Reply:
I _think_ these small logos are okay, since we have prior art in the legacy OBW... it was the large 1MB + images used in the homepage templates that we need to watch out for.